build/python.props

```diff
@@ -1,5 +1,5 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="1.1.202005032" />
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="3.0.13905" />
```

Comment:
Are making a major version bump here?

Reply:
Assuming this is the new build generating nuget package with following semver semantics?

Reply:
Followed up offline. We can address later.

Reply:
@pragnagopa it doesn't match the release tags for the actual worker. This is artificially aligning with the host